addons/mail/static/src/core_ui/thread.xml

```diff
@@ -26,7 +26,7 @@
                         </div>
                         <t t-set="currentDay" t-value="msg.dateDay"/>
                     </t>
-                    <Transition visible="props.thread.seen_message_id === (prevMsg?.id ?? false)" name="'o-fade'" t-slot-scope="transition">
+                    <Transition t-if="props.thread.isChannel"  visible="props.thread.seen_message_id === (prevMsg?.id ?? false)" name="'o-fade'" t-slot-scope="transition">
```

Comment:
We should remove `isChannel` getter and just use `.model === 'mail.channel'` (Soon becoming `discuss.channel` 😎)